api/v1alpha1/oadp_types.go

```diff
@@ -232,8 +233,18 @@ type SnapshotLocation struct {
 	Velero *velero.VolumeSnapshotLocationSpec `json:"velero"`
 }
 
+// NonAdmin defines the configuration for the non admin feature
+type NonAdmin struct {
+	// Enables non admin feature, by default false
```

Comment:
I would re-frame to say `by default disabled` and not `by default false` because `nil` behavior in our case means `disabled` , right ?

Reply:
correct, I will make the change